DependencyInjection/Configuration.php

```diff
@@ -37,6 +37,13 @@ public function getConfigTreeBuilder(): TreeBuilder
                         ->scalarNode('sender_name')->isRequired()->cannotBeEmpty()->end()
                     ->end()
                 ->end()
+                ->arrayNode('password')
+                ->isRequired()
+                    ->children()
+                        ->scalarNode('minStrenght')->isRequired()->cannotBeEmpty()->end()
+                        ->scalarNode('minLenght')->isRequired()->cannotBeEmpty()->end()
```

Comment:
```suggestion
                        ->scalarNode('min_strength')->isRequired()->setDefaultValue(4)->end()
                        ->scalarNode('min_length')->isRequired()->setDefaultValue(8)->end()
```

Reply:
Als we hem altijd willen weten of fallback variabele willen instellen dan kan dat op deze manier. Een andere manier is om het via een flex recipe al in de config te zetten.

Reply:
+ geen camelCase in config waardes